packages/protocol/scripts/foundry/create_and_migrate_anvil_l2_devchain.sh

```diff
@@ -31,9 +31,10 @@ CELO_DISTRIBUTION_SCHEDULE_ADDRESS=$(
 # implement the receive function nor does it allow ERC20 transfers. This is the only way I 
 # managed to give the CeloUnreleasedTreasure a balance.
 echo "Setting CeloUnreleasedTreasure balance..."
+HEX_CELO_DISTRIBUTION_SCHEDULE_INITIAL_BALANCE=$(cast to-hex $CELO_DISTRIBUTION_SCHEDULE_INITIAL_BALANCE"000000000000000000")
```

Comment:
might be a good opportunity to also rename the variable to `CELO_UNRELEASED_TREASURY_INITIAL_BALANCE`